app/models/invenio_rdm_record_converter.rb

```diff
@@ -80,7 +88,9 @@ def invenio_metadata(gf)
       "creators": creators(gf.creator),
       "title": gf.title.first,
       "additional_titles": gf.title.last(gf.title.size-1).map{ |title| {"title": title, "type": "alternative_title", "lang": "eng"} },
-      "formats": gf.mime_type
+      "subjects": SUBJECT_SCHEMES.map{ |subject_type| subjects_for_scheme(gf.send(subject_type), subject_type) }.flatten,
+      "formats": gf.mime_type,
+      "locations": gf.based_near.present? ? gf.based_near.shift.split("', ").map{ |location| {place: location.gsub("'", "")} } : {}
```

Comment:
This was surprisingly annoying because of how the locations are entered. This gets the job done though even if it's ugly.

Reply:
FWIW I tested with both a single location and two locations, but I left it with two since splitting a string on a non-existent sub-string will return the full string in an array. In other words, `"some place, Illinois, USA".split("', ")` will return `["some place, Illinois, USA"]` which is then iterated over with map as expected.

Reply:
Blech, that's how `based_near` stores it's values? Sorry, but thanks for dealing with it.